Description: Set default path for scratchbox sb2_config tool
 The scratchbox2 package in Debian installs the sb2_config tool in /usr/bin.
 This tool is used by the anjuta scratchbox plugin to configure the scratchbox
 login. As the plugins searches for $(build_path)/bin/sb*_config, set the
 default value for build_path to "/usr".
 Also set default version to Sbox2, as scratchbox2 is the only available
 version in the Debian archive.
Index: anjuta-extras-3.4.0/plugins/scratchbox/org.gnome.anjuta.plugins.scratchbox.gschema.xml.in
===================================================================
--- anjuta-extras-3.4.0.orig/plugins/scratchbox/org.gnome.anjuta.plugins.scratchbox.gschema.xml.in	2012-03-18 23:51:40.000000000 +0100
+++ anjuta-extras-3.4.0/plugins/scratchbox/org.gnome.anjuta.plugins.scratchbox.gschema.xml.in	2012-03-30 01:35:31.765791129 +0200
@@ -1,14 +1,14 @@
 <schemalist>
 	<schema id="org.gnome.anjuta.plugins.scratchbox" path="/org/gnome/anjuta/plugins/scratchbox/">
 		<key name="build-path" type="s">
-			<default>"/scratchbox"</default>
+			<default>"/usr"</default>
 		</key>
 		<key name="version" type="s">
 			<choices>
 				<choice value="Sbox1" />
 				<choice value="Sbox2" />
 			</choices>
-			<default>"Sbox1"</default>
+			<default>"Sbox2"</default>
 		</key>
 		<key name="target" type="s">
 			<default>"None"</default>
